e2e-tests/playwright/e2e/verify-redis-cache.spec.ts

```diff
@@ -37,38 +56,34 @@ test.describe("Verify Redis Cache DB", () => {
       timeout: 60_000,
     });
 
-    // Wait for port-forward to be ready
-    await new Promise<void>((resolve, reject) => {
-      portForward.stdout.on("data", (data) => {
-        if (data.toString().includes("Forwarding from 127.0.0.1:6379")) {
-          resolve();
-        }
-      });
-
-      portForward.stderr.on("data", (data) => {
-        reject(new Error(`Port forwarding failed: ${data.toString()}`));
-      });
-    });
-
-    const redis = new Redis(
-      `redis://${process.env.REDIS_TEMP_USER}:${process.env.REDIS_TEMP_PASS}@localhost:6379`,
+    console.log("Connecting to Redis...");
+    redis = new Redis(
+      `redis://${process.env.REDIS_USERNAME}:${process.env.REDIS_PASSWORD}@localhost:6379`,
     );
+    console.log("Verifying Redis keys...");
     await expect(async () => {
-      const keys = await redis.keys("*");
+      const keys = (await redis.keys("*")).filter((k) =>
+        k.includes("techdocs"),
+      );
       expect(keys).toContainEqual(expect.stringContaining("techdocs"));
-
-      // Additionally, verify the format of the key
-      const key = keys.filter((k) => k.startsWith("techdocs"))[0];
+      const key = keys[0];
+      console.log(`Verifying key format: ${key}`);
       expect(key).toMatch(
-        /(?:techdocs|bulk-import):(?:[A-Za-z0-9+]{4})*(?:[A-Za-z0-9+]{2}==|[A-Za-z0-9+]{3}=)$/gm,
+        /(?:techdocs|bulk-import):(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{4})$/gm,
```

Comment:
Extend to match cache entries with `/` like this one: `techdocs:Vu4FxaRdm8ntmAMluyzTCjxOX6QMAIwTwTK/hg7zzgg=`